src/error.rs

```diff
@@ -79,11 +79,12 @@ impl From<Infallible> for ToBytesError {
 
 impl fmt::Display for ToBytesError {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        use ToBytesError::*;
+        use ToBytesError as E;
 
         match *self {
-            InvalidChar(ref e) => write_err!(f, "invalid char, failed to create bytes from hex"; e),
-            OddLengthString(ref e) =>
+            E::InvalidChar(ref e) =>
+                write_err!(f, "invalid char, failed to create bytes from hex"; e),
+            E::OddLengthString(ref e) =>
```

Comment:
IMO the problems the original way causes are too unlikely to bother but I'm also not going to change it back.

Reply:
I also don't like the original way stylistically. It's hard to tell where the symbols are coming from because of the wildcard import.

Reply:
OK, though note that for me this, as opposed to other wildcard cases, is an easily-recognizable pattern. The `match` is matching on what's clearly an enum, based on its structure and if the variants don't have the `X::` prefix then it's clear it must be wildcard imported in the statement above match. Of course one could uselessly import a different enum but that would cause other warnings. It does require a bit of discipline - to only put the `use` in the same function. But I have no problem with such discipline.

Anyway, not a big deal.

Reply:
As everyone knows I'm a sucker for uniformity. Sometimes the variant names clash and then the wildcard cannot be used which means breaking uniformity. The new style is always valid _and_ stomps the lint warning - seems like a win. It is admittedly a bit ugly.